external/fv3fit/tests/emulation/thermobasis/test_emulator.py

```diff
@@ -152,6 +152,7 @@ def test_Config_register_parser(args, loss_cls):
 
 
 @given(lists(integers(min_value=0, max_value=100)))
+@settings(deadline=None)
 def test_Config_multi_output_levels(levels):
     str_levels = ",".join(str(s) for s in levels)
```

Comment:
I think this test may also fail when `levels` has a duplicate entry.  Can you also add this?
```suggestion
    str_levels = ",".join(str(s) for s in set(levels))
```